### PR TITLE
chore(docker): download mlocati/docker-php-extension-installer

### PR DESCRIPTION
**Pull request by dunglas**
Prevents an annoying warning in our CI (which is a false positive).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN set -eux; \
 	echo '<?php phpinfo();' > /app/public/index.php
 
 COPY --link caddy/frankenphp/Caddyfile /etc/caddy/Caddyfile
-COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+ADD --chmod=0755 https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
```
Comment (withinboredom) [inline Dockerfile:27]: Won't this mean that every build step beyond here will be uncacheable?

Comment (withinboredom) [reply Dockerfile:27]: In other words, maybe we should make this one of the very last steps?

Comment (dunglas) [reply Dockerfile:27]: Indeed

Comment (dunglas) [reply Dockerfile]: Actually, it's already the last (not instant) step of this stage

**Pull request merged**